public/home.html

```diff
@@ -131,6 +139,30 @@
         <div class="col-xs-12 col-md-2">
           <button v-on:click="edit(doc._id)" class="btn btn-default">Edit</button>
         </div>
+        <div v-if="showNotes" class="col-md-12 text-muted">
+          <div class="panel panel-default">
+            <div class="panel-heading">Notes</div>
+            <div class="panel-body">
+              <div class="row">
+                <div class="col-md-12">
+                  <table class="table table-condensed">
+                    <tr v-if="doc.notes">
+                      <th colspan="6">what</th>
+                      <th>who</th>
+                      <th>when</th>
+                    </tr>
+                    <tr v-for="note in doc.notes">
+                      <td colspan="6">{{ note.note }}</td>
+                      <td>{{ note.who }}</td>
+                      <td>{{ note.time }}</td>
```

Comment:
maybe we can make the datetime a littl more user friendly:
```
<td>{{ new Date(note.time).toLocaleString() }}</td>
```
i would suggest doing the same in the Edit page as well